kaggle_environments/envs/lux_ai_s3/lux_ai_s3.py

```diff
@@ -0,0 +1,164 @@
+import dataclasses
+import json
+import math
+import os
+import random
+import sys
+from os import path
+from .agents import all_agents
+from sys import path as syspath
+from os import path as osp
+
+# next two lines enables importing local packages e.g. luxai_s2
+__dir__ = osp.dirname(__file__)
+syspath.append(__dir__)
+
+
+# import vec_noise
```

Comment:
this needs to be imported inside the functions because our visualizer environment (cloud functions) doesn't support vec-noise package.

So if you import it within a function you can still use it and the replay won't invoke it.

Reply:
actually vec noise isn't used, using a jax based noise function (which is naturally vectorized via vmap)

i can remove some of the old code/commented out parts from S2